production/aggregator/application.yaml

```diff
@@ -15,26 +15,34 @@ spec:
           value: |
             image:
               repository: europe-west6-docker.pkg.dev/ctdapp-391309/ctdapp/cover-traffic
-              tag: "0195a42"
-            replicaCount: 1
-            args: 
+              tag: "2a8d28b"
+            replicaCount: 0
```

Comment:
Disable?

Reply:
For production yes it should be set to 0 until we have our staging version finalised